rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/with_value_as_argument.php.inc

```diff
@@ -9,7 +9,7 @@ final class WithValueAsArgument
      * @When
      * @When "this value is within quotes"
      * @When this value has a ' character
-     * @When(key="value") this annotation has parameters so won't use this option
```

Comment:
I think the description should be kept as comment when exists, when there is description outside `()`, which currently cause diff:

```diff
-    #[\Behat\Step\When('(key="value")')]
+    #[\Behat\Step\When('(key="value")this annotation has parameters so won\'t use this option')]
```

which should be:

```diff
-    #[\Behat\Step\When('(key="value")')]
+    #[\Behat\Step\When('(key="value")')] // this annotation has parameters so won't use this option
```

I will look into it when possible.

Reply:
I haven't seen this case in any codebase yet.

Reply:
covered at https://github.com/rectorphp/rector-src/pull/6589